cpp/modmesh/onedim/Euler1DCore.hpp

```diff
@@ -107,12 +108,18 @@ class Euler1DCore
     SimpleArray<double> const & gamma() const { return m_gamma; }
     SimpleArray<double> & gamma() { return m_gamma; }
 
-    double density(size_t it) const { return m_so0(it, 0); }
+    inline double density(size_t it) const { return m_so0(it, 0); }
```

Comment:
Please remove the inlines.  They are not effective.  Code in the class declaration is automatically inline.

Reply:
I remove these unnecessary inline in https://github.com/solvcon/modmesh/pull/210/commits/b2b9a86d99b8001789636595f82f2114586d8261